mazda/outputs.cpp

```diff
@@ -107,6 +137,7 @@ void VideoOutput::input_thread_func()
         FD_ZERO(&set);
         FD_SET(touch_fd, &set);
         FD_SET(kbd_fd, &set);
+        FD_SET(ui_fd, &set);
```

Comment:
I don't think you want to add this here. Since it's write-only the select will wake up if there is buffer to write but since it's also marked as non-blocking I'm not sure if this is valid. Plus the code never checks if that fd is set so it seems unused. If you do need to select on it, you also need to add it to the maxfdPlus1 calculation due to the insane way this API is designed :)